hoomd/data.py

```diff
@@ -31,28 +31,32 @@
 
 Examples::
 
+    import hoomd
+    system = hoomd.init.create_lattice(unitcell=hoomd.lattice.sc(a=2.0), n=5);
     snapshot = system.take_snapshot()
     system.restore_snapshot(snapshot)
-    snapshot = data.make_snapshot(N=100, particle_types=['A', 'B'], box=data.boxdim(L=10))
+    snapshot = hoomd.data.make_snapshot(N=100, particle_types=['A', 'B'], box=data.boxdim(L=10))
     # ... populate snapshot with data ...
-    init.read_snapshot(snapshot)
+    hoomd.init.read_snapshot(snapshot)
 
 .. rubric:: Snapshot and MPI
 
 In MPI simulations, the snapshot is only valid on rank 0 by default. make_snapshot, read_snapshot, and take_snapshot,
 restore_snapshot are collective calls, and need to be called on all ranks. But only rank 0 can access data
 in the snapshot::
 
+    import hoomd
+    system = hoomd.init.create_lattice(unitcell=hoomd.lattice.sc(a=2.0), n=5);
     snapshot = system.take_snapshot(all=True)
     if comm.get_rank() == 0:
```

Comment:
Please replace `comm.get_rank()` with `hoomd.comm.get_rank()` in all examples you modified